delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/model/FileEvent.scala

```diff
@@ -133,6 +137,8 @@ object FileEvent {
   final case class FileAttributesUpdated(
       id: Iri,
       project: ProjectRef,
+      storage: ResourceRef.Revision,
```

Comment:
We can also add them in `FileEvent` itself

Reply:
Thanks, fixed.